mod/pages/views/default/pages/input/parent.php

```diff
@@ -1,37 +1,6 @@
 <?php
 /**
- * Parent picker
- *
- * @uses $vars['value']          The current value, if any
- * @uses $vars['options_values']
- * @uses $vars['name']           The name of the input field
- * @uses $vars['entity']         Optional. The child entity (uses container_guid)
+ * @depreated 3.0 Use input/pages/parent
  */
 
```

Comment:
maybe also add a deprecation notice so devs actually change the use of this view